packages/frontend/src/components/common/Table/ScrollableTable/TableHeader.tsx

```diff
@@ -15,12 +16,42 @@ import { HeaderDndContext, HeaderDroppable } from './HeaderDnD';
 
 interface TableHeaderProps {
     minimal?: boolean;
+    showSubtotals?: boolean;
 }
 
-const TableHeader: FC<TableHeaderProps> = ({ minimal = false }) => {
+const TableHeader: FC<TableHeaderProps> = ({
+    minimal = false,
+    showSubtotals = true,
+}) => {
     const { table, headerContextMenu, columns } = useTableContext();
     const HeaderContextMenu = headerContextMenu;
     const currentColOrder = React.useRef<Array<string>>([]);
+
+    useEffect(() => {
+        if (showSubtotals) {
+            const groupedColumns: any = columns
+                .filter(
+                    (col: any) =>
+                        col.meta?.item?.fieldType === FieldType.DIMENSION,
+                )
+                .map((col: any) => col.id);
+            const sortedColumns: any = table
+                .getState()
+                .columnOrder.reduce((acc: any[], sortedId) => {
```

Comment:
This code makes sense, however, it would be nice if we could avoid having `ANY` as it often introduces unnecessary runtime bugs. I'm happy to chime in here and update types.

Reply:
I got rid of the `ANY`'s but not sure my code to replace them is what you have in mind. Please let me know.